packages/react/src/__tests__/ReactElementJSX-test.internal.js

```diff
@@ -34,6 +35,7 @@ describe('ReactElement.jsx', () => {
     ReactFeatureFlags.warnAboutSpreadingKeyToJSX = true;
```

Comment:
If this test wasn't marked as "internal", it would fail on bundles as it should — because we haven't exposed these entry points on npm. To do that, check out how files in `npm/` folders in different packages work. Also note `"files"` array in `package.json` of each package.

Reply:
https://github.com/facebook/react/pull/18301